src/views/box_view.rs

```diff
@@ -212,12 +213,25 @@ impl<T: View> BoxView<T> {
 impl<T: View> ViewWrapper for BoxView<T> {
     wrap_impl!(self.view: T);
 
+    fn wrap_draw(&self, printer: &Printer) {
+        let mut printer = printer.clone();
+        // Set values hard here, since cropping would allow smaller size if fixed or at least set
+        printer.size.x = self.size.x.result((printer.size.x, printer.size.x));
+        printer.size.y = self.size.y.result((printer.size.y, printer.size.y));
```

Comment:
You could use `Printer::cropped` and `Vec2::zip_map`:
```rust
self.view.draw(&printer.cropped(
    self.size.zip_map(printer.size, |c, s| c.result((s, s)))
));
```

Reply:
Good suggestion, used the `zip_map` in the new commit. But using the `cropped` of the printer doesn't work at the moment e.g. the PanelView as I used in the example still prints the full panel with the size of the printer. But `inner_size` is applicable for in this situation.

Reply:
Ah right sorry - you'll want `printer.inner_size()` in this case, since we're only changing the size visible to the child.